addons/account/models/template_generic_coa.py

```diff
@@ -17,7 +17,7 @@ def _get_generic_coa_template_data(self):
         :rtype: dict
         """
         return {
-            'name': "Generic Chart Template",
+            'name': "United States of America (Generic)",
```

Comment:
```suggestion
            'name': _("United States of America (Generic)"),
```
Shouldn't it be translated?
If so, please include the translations :wink:

Reply:
i can add the entry in the pot :smile: i will do it now

Reply:
It make clearly too much translation on the pot ahah, i will just add the translation entry of this, tell me what you think :smile: